src/locales/en-us/common/index.yaml

```diff
@@ -45,14 +45,15 @@ common:
     text1: 'Are you sure you want to delete the {type} {name}?'
     text2: 'This action cannot be reversed.'
     title: 'Delete {type}'
-  tray:
-    close: 'Close'
   emptyState:
     title: 'No data'
     message: 'There are no {type} present'
     icon: 'stateNoData'
   collection:
     none: ''
+    tray:
+      empty_title: 'Couldn’t find the selected item'
+      empty_message: 'The selected item either doesn’t exist or isn’t on the current page.'
```

Comment:
Feel like we could use `type` here instead of 'item', similar to the code in the `emptyState` above, again small detail, doesn't necessarily need to happen here although it would probably help with the rollout if we did it here.

Reply:
I’ve changed this now.

Reply:
👍